crates/matrix-sdk/src/widget_api/messages/capabilities.rs

```diff
@@ -91,31 +35,26 @@ impl Serialize for Options {
             capability_list.push("io.element.requires_client".to_owned());
         }
 
-        if let Some(ev_filter) = &self.send_room_event {
-            for event_type in ev_filter {
-                let filter = serde_json::to_string(&event_type).unwrap();
-                capability_list.push(format!("org.matrix.msc2762.m.send.event{}", filter));
-            }
+        for event_type in &self.send_room_event {
+            let filter = serde_json::to_string(event_type).unwrap();
```

Comment:
Is it safe to `unwrap()`, I think if something can fail, we must return an error, luckily the function also returns an error.

Reply:
no unsae unwraps in this file anymore. (only `serde_json::to_value("").unwrap()` is left which never fails (or always ;) )

Reply:
tested: should never fail